src/platform/windows/window.rs

```diff
@@ -951,7 +953,7 @@ unsafe fn init(
         };
 
         if !attributes.resizable {
-            style &= !winuser::WS_SIZEBOX;
+            style &= !WS_RESIZABLE as u32;
```

Comment:
I believe the type is `DWORD`, not `u32`.

Reply:
K thx, my IDE just told me it was u32 which is probably the same thing.